pkg/build/types/types.go

```diff
@@ -212,6 +212,9 @@ func ParseArchitecture(s string) Architecture {
 func ParseArchitectures(in []string) []Architecture {
 	uniq := map[Architecture]struct{}{}
 	for _, s := range in {
+		if s == "all" {
```

Comment:
Is the intention to be able to support --arch=x86,all? That seems a little weird.

I think in ko we only allow "all" if it's the only element.

Reply:
Yes, and that emits an warning if im not wrong